tests/foreman/api/test_contentview.py

```diff
@@ -293,6 +317,85 @@ def test_positive_publish_2(self):
             cvv = entities.ContentViewVersion(id=cvv_id)
             self.assertGreater(cvv.read_json()['package_count'], 0)
 
+    def test_publish_composite_cv_once_1(self):
+        """@Test: Create empty composite view and assign one normal content
+        view to it. After that publish that composite content view once.
+
+        @Assert: Composite content view is published and corresponding
+        version is assigned to it.
+
+        @Feature: ContentView
+
+        """
+        composite_cv = entities.ContentView(
+            composite=True,
+            organization=self.org.id,
+        ).create()
+        self.add_content_views_to_composite(composite_cv)
+        composite_cv.publish()
+        self.assertEqual(len(composite_cv.read_json()['versions']), 1)
+
+    def test_publish_composite_cv_once_2(self):
+        """@Test: Create empty composite view and assign random number of
+        normal content views to it.
```

Comment:
Also, if using vim, you can issue this handy sequence of commands:

    set textwidth=79
    gqap←Line wrap around paragraph.
    gq}←Line wrap to end of paragraph.
    gqj←Text wrap this line and the one below it.
    gq2j←Text wrap this line and two below it.